### PR TITLE
Added support for getParameterFormDescs in AEmbeddedRanking

### LINKED ISSUE #228: AEmbeddedRanking does not support getParameterFormDescs to show a menu bar
Usually subclasses of AView are able to override the method `getParameterFormDescs` which returns an `IFormElement[]` such that form elements are rendered above the ranking.

With AEmbeddedRanking this is currently not possible.

## Is this change breaking?

No, should not be breaking

### PR DESCRIPTION
**Pull request by lehnerchristian**
closes #228 

this PR adds support for the `getParameterFormDescs` method as available in `AView`.

AEmbeddedRanking is used as a wrapper around an `ARankingView` (AEmbeddedRanking instantiates an `EmbeddedRanking` which inherits from `ARankingView`) to be able to embed it directly at the start of an application workflow (without inputSelection, as opposed to a detail view --> like the RankingView in Ordino)

Long story short: AEmbeddedRanking contains a nested class `EmbeddedRanking` which inherits from `ARankingView`, however `getParameterFormDescs` is not forwarded to the nested class.

This PR adds the `getParameterFormDescs` method to AEmbeddedRanking and calls it in the overriddden `getParameterFormDescs` from `EmbeddedRanking`

### CODE REVIEW

File: src/base/AEmbeddedRanking.ts [hunk 1]
```diff
@@ -105,6 +106,14 @@ export abstract class AEmbeddedRanking<T extends IRow> implements IViewProvider
       runWithoutTracking<T>(f: () => T): Promise<T> {
         return super.withoutTracking(f);
       }
+
+      protected getParameterFormDescs() {
```
Comment (thinkh) [inline src/base/AEmbeddedRanking.ts]:
```suggestion
      protected getParameterFormDescs(): IFormElementDesc[] {
```

I would type the return value.

File: src/base/AEmbeddedRanking.ts [hunk 2]
```diff
@@ -105,6 +106,14 @@ export abstract class AEmbeddedRanking<T extends IRow> implements IViewProvider
...
+        const base = super.getParameterFormDescs();
+        return [
+          ...base,
+          ...that.getParameterFormDescs()
```
[the remark this answers is not in the document]
Comment (lehnerchristian) [reply src/base/AEmbeddedRanking.ts]: it's defined right above the `AEmbeddedRankingView`. it is also used for `loadColumnDesc`

[the remark this answers is not in the document]
Comment (thinkh) [reply src/base/AEmbeddedRanking.ts]: Ok, that part was folded in the Github code view.

**Comment by thinkh** [comment]
@lehnerchristian I assume this PR will cause a minor release, right? Other than that you can go ahead and merge the PR.

**Comment by lehnerchristian** [comment]
yes, it'll be minor. thanks for reviewing :+1:
